build.gradle

```diff
@@ -315,6 +315,11 @@ dependencies {
 
   implementation group: 'org.slf4j', name: 'slf4j-api', version: versions.slf4j
 
+  implementation group: 'com.azure', name: 'azure-core', version: '1.13.0'
```

Comment:
Should we be using more up to date versions? Could you also please move these into the versions block above to keep everything consistent?